ui/apps/platform/src/Containers/ComplianceEnhanced/ClusterCompliance/Coverage/Clusters/ClusterDetailsTable.tsx

```diff
@@ -132,26 +143,30 @@ function ClusterDetailsTable({
     }
 
     const renderTableContent = () => {
-        return scanResults?.checkResults.map(({ checkName, rationale, clusters }) => {
-            const scanConfigUrl = generatePath(complianceEnhancedScanConfigDetailPath, {
-                scanConfigId: checkName,
-            });
-
+        return scanResults?.checkResults.map((checkResult) => {
+            const { checkName, rationale, clusters } = checkResult;
             const status = getStatusByClusterId(clusters);
             const statusObj = status ? getClusterResultsStatusObject(status) : null;
 
             return (
                 <Tr key={checkName}>
                     <Td modifier="truncate">
                         <>
-                            <Link to={scanConfigUrl}>{checkName}</Link>
-                            <br />
-                            <small className="pf-u-color-200">{rationale}</small>
+                            <Text>{checkName}</Text>
+                            <Text component={TextVariants.small} className="pf-u-color-200">
```

Comment:
nit: Mark pointed out that, in TS files, we can just use the bare strings and TS will catch if we mis-type the constant names allowed for PF props.

Reply:
I feel like there's still a benefit to using the enum. Easier to find and potentially replace certain modals that use a specific variant. I'd hope patternfly also adds the enums so that they have the option of changing the value without forcing us to update our code everywhere.